packages/openapi-generator/src/wrapper-files/create-file.ts

```diff
@@ -41,3 +41,21 @@ ${content}
 ` + '\n'
   );
 }
+
+/**
+ * @experimental This API is experimental and might change in newer versions. Use with caution.
+ * Copy a file from a given path.
+ * @param fromPath Path of the original file to be copied.
+ * @param toFileName File name of the new file.
+ * @param toDirectoryPath Path of the directory to write to.
+ * @param overwrite Whether or not existing files should be overwritten.
+ */
+export async function copyFile(
+  fromPath: string,
+  toFileName: string,
+  toDirectoryPath: string,
+  overwrite: boolean
+): Promise<void> {
+  const fileContent = readFileSync(resolve(fromPath), { encoding: 'utf8' });
```

Comment:
I would really like to avoid using sync calls where possible. The async methods don't block the execution and I think we can boost our performance with this (especially when generating many services).
Also, why don't we use the fs.copyFile method?